roslibrust_ros1/Cargo.toml

```diff
@@ -32,6 +32,7 @@ regex = { version = "1.9" }
 byteorder = "1.4"
 thiserror = "2.0"
 anyhow = "1.0"
+if-addrs = "0.14.0"
```

Comment:
`get_if_addrs` is explicitly a non-windows function in the if_addrs crate. Which means adding this MR explicitly makes `roslibrust` not support windows. While we haven't been explicitly testing for or advertising windows support, I actually do expect it to work across all our backends at the moment.

Looking around it looks like the `getifs` crate may be a better alternative to grab that does support windows?